tests/test_create_account.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import re
+
+import pytest
+from unittestzero import Assert
+
+from fxapom.fxapom import FxATestAccount
+
+
+@pytest.mark.nondestructive
```

Comment:
The accounts are being created via the test account server, so I kind of see them as nondestructive, but I see your point. Removing the mark will still allow the tests to run against the dev environment.